client/app/card/card.component.ts

```diff
@@ -199,15 +199,6 @@ export class CardComponent extends NaturalAbstractController implements OnInit,
         ...quillConfig.modules,
         keyboard: {
             bindings: {
-                enter: {
```

Comment:
@PowerKiKi was this change intentional ?

Reply:
No, that was most likely a copy-paste mistake from d17d3dfb9067e4286a8a7ec8c74912a2d29b06af. IMHO this PR should be merged.